src/components/modal/v2/parts/Header.jsx

```diff
@@ -40,7 +41,7 @@ const Header = ({
                 <div className="header__background-wrapper header__background-wrapper--gradient" />
```

Comment:
```suggestion
                <div
                    className={`header__background-wrapper header__background-wrapper--gradient ${
                        useV4Design ? 'v4Design' : ''
                    }`}
                />
```

Reply:
updated Header.jsx. I don't think _header.scss needs to be updated because of the nesting.